Core/LinkCleaner.swift

```diff
@@ -88,4 +88,42 @@ public class LinkCleaner {
         
         return nil
     }
+    
+    public func cleanTrackingParameters(initiator: URL?, url: URL?,
+                                        config: PrivacyConfiguration = PrivacyConfigurationManager.shared.privacyConfig) -> URL? {
```

Comment:
This seems like a good candidate for some unit tests, that test border conditions, config exceptions etc).

Reply:
There is a test suite in the privacy reference tests that is included with this PR.